build.xml

```diff
@@ -119,37 +117,45 @@
         <pathelement location="${basedir}/lib/sqltool.jar"/>
     </path>
 
-    <path id="library.jsoup.classpath">
-        <pathelement location="${basedir}/lib/jsoup-1.10.2.jar"/>
+    <path id="library.pf4j-2.0.0.classpath">
+        <pathelement location="${basedir}/lib/pf4j-2.0.0.jar"/>
     </path>
 
-    <path id="library.openqa-selenium.classpath">
-        <pathelement location="${basedir}/lib/client-combined-3.4.0-nodeps.jar"/>
-    </path>
+    <!-- Libraries used by the plugins -->
 
-    <path id="library.commons-cli.classpath">
-        <pathelement location="${basedir}/lib/commons-cli-1.4.jar"/>
+    <!-- MiBand -->
+    <path id="library.jackson">
+        <path location="${plugins.importer}/miband/lib/jackson-core-2.9.4.jar"/>
+        <path location="${plugins.importer}/miband/lib/jackson-databind-2.9.4.jar"/>
+        <path location="${plugins.importer}/miband/lib/jackson-annotations-2.9.0.jar"/>
     </path>
 
-    <path id="library.google.classpath">
-        <pathelement location="${basedir}/lib/google-api-client-1.23.0.jar"/>
-        <pathelement location="${basedir}/lib/google-api-client-jackson2-1.23.0.jar"/>
-        <pathelement location="${basedir}/lib/google-api-services-fitness-v1-rev30-1.23.0.jar"/>
-        <pathelement location="${basedir}/lib/google-api-services-people-v1-rev213-1.23.0.jar"/>
-        <pathelement location="${basedir}/lib/google-http-client-1.23.0.jar"/>
-        <pathelement location="${basedir}/lib/google-http-client-jackson2-1.23.0.jar"/>
-        <pathelement location="${basedir}/lib/google-maps-services-0.2.6.jar"/>
-        <pathelement location="${basedir}/lib/google-oauth-client-1.23.0.jar"/>
-        <pathelement location="${basedir}/lib/google-oauth-client-java6-1.23.0.jar"/>
-        <pathelement location="${basedir}/lib/google-oauth-client-jetty-1.23.0.jar"/>
-        <pathelement location="${basedir}/lib/jsr305-1.3.9.jar"/>
+    <!-- Medtronic Crawler-->
+    <path id="library.jsoup.classpath">
+        <pathelement location="${plugins.importer}/medtroniccrawler/lib/jsoup-1.10.2.jar"/>
     </path>
 
-    <path id="library.jfree-chart.classpath">
-        <pathelement location="${basedir}/lib/jcommon-1.0.23.jar"/>
-        <pathelement location="${basedir}/lib/jfreechart-1.0.19.jar"/>
+    <!-- Google Crawler -->
```

Comment:
warum so umständlich, sag doch in jedem plugin das die jeweiligen libs immer unter lib/ bezüglich des packages zu finden sind

Reply:
Uhm Probleme beim compilieren, muss ja auch dem Compiler sagen, wo die Libs liegen
Aber mir fällt gerade auf, dass der plugin Task ja auch den javac nutzt, von daher mal schauen, ob ich dem da die Libs mitgeben kann